### ISSUE TITLE
Buggy Behavior in Android WebView

### ISSUE THREAD

**Issue by ma7moudat** [manuelstofer/pinchzoom#115]
We're using the same frontend code for the website and the Android/iOS apps. Zoom works fine in all contexts (desktop browsers, mobile browsers, iOS app) except in the Android app (which basically just shows the website inside a WebView).

The Zoom in this case isn't completely unresponsive, but it takes really hard taps to kickstart it and it stops midway while the pinch gesture is taking place for no reason. We tested this on multiple modern and older devices and with Android Studio emulators with same results.

Is this a known issue?
Are there any workarounds?
Could the issue be caused by the usage of the touch-events API instead of the pointer-events API?

Aside from that it works great, so thank you for the nice plugin :)
Appreciate the help 🙏

**Comment by sandstrom** [comment]
Since this code is open-source, without anyone getting paid to work on it, it's difficult to satisfy all the wishes that people may have. I hope you understand! 🌻 

Yeah, I think it would be a good idea to rewrite this library to use pointer events, see https://github.com/manuelstofer/pinchzoom/issues/114

If you can nail down the bug/issue I'm happy to review a patch. Other than that there isn't much I can do.

I'm not even the original author of this library, merely a custodian.

**Comment by ma7moudat** [comment]
It seems to be a WebView issue.

I tried the simplest [pinch gesture example](https://mdn.github.io/dom-examples/pointerevents/Pinch_zoom_gestures.html) from MDN. Same problem; it works fine everywhere except in Android's WebView. So I don't think it can be solved on the JavaScript side.

I'm no app developer, so I'm not sure whether the touch sensitivity of a WebView can be adjusted.

Closing because issue can't be fixed in the library.
